### PR TITLE
Backend: Linear first AREAs

### PR DESCRIPTION
**Pull request by EdenComp**
Also fixes ARE-356

### CODE REVIEW

File: backend/supervisor/jobs/images.go [hunk 1]
```diff
@@ -64,6 +66,8 @@ var OptArgument = map[string]string{
 	"google-on-new-video":                    "on-youtube-video",
 	"google-send-email":                      "send-email",
 	"google-update-signature-email":          "update-signature-email",
+	"linear-create-issue":                    "create-issue",
+	"linear-on-issue-create":                 "",
```
Comment (RezaRahemtola) [inline backend/supervisor/jobs/images.go:70]: no "setup-webhook" param here like in github ?

Comment (EdenComp) [reply backend/supervisor/jobs/images.go:70]: No, with github we have to setup a webhook on each repository we want to listen, with Linear its a global webhook

Comment (EdenComp) [reply backend/supervisor/jobs/images.go:70]: Here I just don't launch any job

Comment (RezaRahemtola) [reply backend/supervisor/jobs/images.go:70]: so why not just remove the line ? To avoid an error while launching the job ?

Comment (EdenComp) [reply backend/supervisor/jobs/images.go]: However the line in the above map is mandatory

**Review by RezaRahemtola — APPROVED** [review]

**Review by RezaRahemtola — APPROVED** [review]

**Pull request merged**